proto/minder/v1/minder.proto

```diff
@@ -4055,6 +4055,12 @@ message RestDataSource {
                     max_len: 1000,
                 }
             ];
+
+            // body_from_field is the field in the input to use as the body.
+            // If the value is an string, it will be used as the body, as is.
+            // If the value is an object, it will be serialized as JSON.
+            // If the value is not found in the input, the request will fail.
+            string body_from_field = 10;
```

Comment:
Can we set validation on it? Although I see it's a bit of a mixed use for this field. Would it be better to use oneof?

Reply:
It already is within a `oneof`

Reply:
let me add validatoin